config/output_schemas/full_responses_mapping_qa.toml

```diff
@@ -0,0 +1,580 @@
+[reference]
```

Comment:
we don't need this one, please delete this schema. Check we have the one that's used in the mapping output